From: Sjoerd Simons <sjoerd@luon.net>
Date: Sun, 13 Jul 2014 16:56:16 +0200
Subject: man: sysusers.d correct default user shell

For the non-root user sysusers uses nologin as the default shell, not
login. Correct the documentation to match the code.

(cherry picked from commit eb34cba76323ea673471e226f97721bd6bdbcbf6)
---
 man/sysusers.d.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/sysusers.d.xml b/man/sysusers.d.xml
index 00eb7ec..1e079b2 100644
--- a/man/sysusers.d.xml
+++ b/man/sysusers.d.xml
@@ -102,7 +102,7 @@ m authd input</programlisting>
                                         group will be set to the group
                                         bearing the same name. The
                                         user's shell will be set to
-                                        <filename>/sbin/login</filename>,
+                                        <filename>/sbin/nologin</filename>,
                                         the home directory to
                                         <filename>/</filename>. The
                                         account will be created
